readthedocs/__init__.py

```diff
@@ -0,0 +1,5 @@
+from __future__ import absolute_import
+
+# This will make sure the app is always imported when
+# Django starts so that shared_task will use this app.
+from .settings.celery import app as celery_app  # noqa
```

Comment:
Sounds good. I might create an `core/apps.py` in another PR for CORS here, so we can build on top of that.